frontend/src/pages/calculator/CalculatorElements.js

```diff
@@ -0,0 +1,202 @@
+import { Form, Slider, InputNumber } from 'antd';
+import { SubmitButton } from '../../components/gradientButton/GradientButton';
+import 'antd/dist/antd.css';
+
+export const FormContainer = styled(Form)`
+  height: auto;
+  min-height: 90vh;
+  padding-top: 10vh;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-template-rows: repeat(4, 1fr) 0.5fr;
+  column-gap: 0.5rem;
+  row-gap: 2rem;
+  background-color: #f9f6fb;
+  align-content: center;
+  justify-items: center;
+`;
+export const CustomSlider = styled(Slider)`
+  width: 95%;
+  .ant-slider-rail {
+    background-color: #ccccd8;
+  }
+  .ant-slider-track {
+    background-color: #5e60ce;
+  }
+  .ant-slider-handle {
+    background-color: white;
+    border-color: #5e60ce;
+  }
+  .ant-slider-dot {
+    background-color: #fff;
+    border-color: #5e60ce;
+  }
+  .ant-slider-rail:hover {
+    background-color: #e5e5f0;
+  }
+  .ant-slider-track:hover {
+    background-color: #06dfc8;
+  }
+  .ant-slider-handle:hover {
+    background-color: #5e60ce;
+    border-color: #5e60ce;
+  }
+  .ant-slider-dot:hover {
+    background-color: #fff;
+    border-color: #5e60ce;
+  }
+  .ant-slider-handle:focus {
+    background-color: #5e60ce;
+    border: #5e60ce;
+  }
+  .ant-slider-track:focus {
+    background-color: #06dfc8;
+  }
+`;
+
+export const CustomInput = styled(InputNumber)`
+  color: #6930c3;
+  font-family: 'Roboto';
+  width: 95%;
+
+  .ant-input-number {
+    border-color: #6930c3;
+  }
+  .ant-input-number-group-addon {
+    background-color: #6930c3;
+    color: #fff;
+    border: 1px solid #6930c3;
+  }
+  .ant-input-number-wrap {
+    border: 2px solid #6930c3;
+  }
+  .ant-input-number-focused {
+    box-shadow: 0 0 0 2px rgba(94, 96, 206, 20%);
+  }
+`;
+
+export const DataPlaceholder = styled.div`
+  background-color: #fff;
+  border-radius: 25px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 1.5rem;
+  width: 25vw;
+  height: 200px;
+`;
+
+export const BoxOne = styled(DataPlaceholder)`
+  grid-area: 1/2/2/5;
+`;
+export const BoxTwo = styled(DataPlaceholder)`
+  grid-area: 2/2/3/5;
+`;
+export const BoxThree = styled(DataPlaceholder)`
+  grid-area: 3/2/4/5;
+`;
+export const BoxFour = styled(DataPlaceholder)`
+  grid-area: 3/5/4/9;
+`;
+export const BoxFive = styled(DataPlaceholder)`
+  grid-area: 3/9/4/12;
+`;
+export const BoxSix = styled(DataPlaceholder)`
+  grid-area: 4/2/5/5;
+`;
+export const BoxSeven = styled(DataPlaceholder)`
+  grid-area: 4/5/5/9;
+`;
+export const ButtonContainer = styled.div`
+  grid-area: 4/9/5/12;
+`;
+export const CalculatorSubmitButton = styled(SubmitButton)`
+  width: 25vw;
+  height: 200px;
+  border-radius: 25px;
+  font-size: 1.125rem;
+`;
+
+export const IllustrationContainer = styled.div`
+  .img {
+    width: 80%;
+    height: auto;
+  }
+  grid-area: 1/5/3/12;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+// import styled from 'styled-components';
```

Comment:
Can we get rid of those commented out lines of code?

Reply:
with pleasure!

Reply:
Thanks resolving